pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions_test.go

```diff
@@ -53,16 +52,40 @@ func TestGCEDiskConflicts(t *testing.T) {
 	}
 	errStatus := framework.NewStatus(framework.Unschedulable, ErrReasonDiskConflict)
 	tests := []struct {
-		pod        *v1.Pod
-		nodeInfo   *framework.NodeInfo
-		isOk       bool
-		name       string
-		wantStatus *framework.Status
+		pod                 *v1.Pod
+		nodeInfo            *framework.NodeInfo
+		name                string
+		preFilterWantStatus *framework.Status
+		wantStatus          *framework.Status
 	}{
-		{&v1.Pod{}, framework.NewNodeInfo(), true, "nothing", nil},
-		{&v1.Pod{}, framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()), true, "one state", nil},
-		{st.MakePod().Volume(volState).Obj(), framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()), false, "same state", errStatus},
-		{st.MakePod().Volume(volState2).Obj(), framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()), true, "different state", nil},
+		{
+			pod:                 &v1.Pod{},
+			nodeInfo:            framework.NewNodeInfo(),
+			name:                "nothing",
+			preFilterWantStatus: framework.NewStatus(framework.Skip),
+			wantStatus:          nil,
+		},
+		{
+			pod:                 &v1.Pod{},
+			nodeInfo:            framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()),
+			name:                "one state",
+			preFilterWantStatus: framework.NewStatus(framework.Skip),
+			wantStatus:          nil,
+		},
+		{
+			pod:                 st.MakePod().Volume(volState).Obj(),
+			nodeInfo:            framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()),
+			name:                "same state",
+			preFilterWantStatus: nil,
+			wantStatus:          errStatus,
+		},
+		{
+			pod:                 st.MakePod().Volume(volState2).Obj(),
+			nodeInfo:            framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()),
+			name:                "different state",
+			preFilterWantStatus: nil,
+			wantStatus:          nil,
+		},
```

Comment:
Add a test case for a pod with a volume that doesn't have restrictions. Just in `TestGCEDiskConflicts` should be enough.